wp-content/themes/core/assets/css/src/admin/block-editor/editor.pcss

```diff
@@ -30,3 +30,23 @@
 		margin: 0;
 	}
 }
+
+/*
+* === WIDEN THE EDIT PAGE SIDEBAR ===
+*/
+
+@media (min-width: 1200px) {
+.interface-complementary-area {
+    width: 350px;
+}
+}
```

Comment:
Please add this to `wp-content/themes/core/assets/css/src/admin/block-editor/editor.pcss`. Additionally swap the block such that the media query is inside the selector:
```
/* Make the sidebar editor wider for large screens. */
.interface-complementary-area {
	@media (min-width: 1200px) {
		width: 350px;
	}
}
```